spec/models/apartment_spec.rb

```diff
@@ -6,4 +6,70 @@
     apartment = user.apartments.create street: "1234 Street Lane", city: 'San Diego', state: 'CA', manager: 'Elyse', email: 'elyse_manager@email.com', price: '3000', bedrooms: 1, bathrooms: 1, pets: '0', image: 'image'
     expect(apartment).to be_valid
   end
+
+  it 'checks a street is inputted' do
+    user = User.where(email: 'test@example.com').first_or_create(password:'12345678', password_confirmation:'12345678')
+    apartment = user.apartments.create city: 'San Diego', state: 'CA', manager: 'Elyse', email: 'elyse_manager@email.com', price: '3000', bedrooms: 1, bathrooms: 1, pets: '0', image: 'image'
+  expect(apartment.errors{:street}).to_not be_empty
```

Comment:
To access errors for a specific key, I would recommend using square brackets instead of curlies.  Ex: [:street] vs {:street}